test/com/google/enterprise/adaptor/filenet/DocumentTraverserTest.java

```diff
@@ -360,6 +361,37 @@ public void testGetDocContent() throws Exception {
   }
 
   @Test
+  public void testGetDocContent_markAllDocsPublic() throws Exception {
+    options = TestObjectFactory.newConfigOptions(
+        ImmutableMap.<String, String>of (
+            "adaptor.markAllDocsAsPublic", "true"));
+
+    String id = "{AAAAAAAA-0000-0000-0000-000000000000}";
+    DocId docId = newDocId(new Id(id));
+    MockObjectStore os = getObjectStore();
+    mockDocument(os, id, DOCUMENT_TIMESTAMP, true, 1000d, "text/plain");
```

Comment:
I'm not wildly persuaded when your mock has no permissions set on it. I get the null vs empty ACL distinction, but perhaps make that explicit by having two tests.